package.json

```diff
@@ -27,9 +28,11 @@
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
     "react-ga": "^3.1.2",
+    "react-markdown": "^4.3.1",
     "react-time-ago": "^6.0.1",
     "stylus": "^0.54.7",
-    "twitter-text": "^3.0.0"
+    "twitter-text": "^3.0.0",
```

Comment:
This dep doesn't seem to be required anymore in this revision of the PR.
```suggestion
```

Reply:
This seems to be used in `components/feed-transformers.ts` (the comma got added after the line)